agents/image-editor-agent/python-server/main.py

```
@@ -0,0 +1,1402 @@
+    # Use PIL for text rendering
+    with Image.open(inputPath) as img:
+        # Convert to RGB if necessary
+        if img.mode != 'RGB':
+            img = img.convert('RGB')
+        
+        draw = ImageDraw.Draw(img)
+        
+        # Try to use a default font, fallback to default if not available
+        try:
+            font = ImageFont.truetype("/System/Library/Fonts/Arial.ttf", fontSize)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)